src/OSPSuite.UI/Services/DialogCreator.cs

```diff
@@ -80,26 +83,44 @@ public ViewResult MessageBoxYesNo(string message, ViewResult defaultButton = Vie
       public ViewResult MessageBoxYesNo(string message, string yes, string no, ViewResult defaultButton = ViewResult.Yes)
       {
          return showQuestionMessageBox(message, new [] {
-            DialogResult.Yes, DialogResult.No}, yes, no, string.Empty, defaultButton);
+            DialogResult.Yes, DialogResult.No}, yes, no, string.Empty, defaultButton, null);
       }
 
-      private ViewResult showQuestionMessageBox(string message, IReadOnlyList<DialogResult> buttons, string yes, string no, string cancel, ViewResult defaultButton)
+      public ViewResult MessageBoxConfirm(string message, Action doNotShowAgain, ViewResult defaultButton = ViewResult.Yes)
+      {
+         return showQuestionMessageBox(message, new[] {
+            DialogResult.Yes, DialogResult.Cancel}, string.Empty, string.Empty, string.Empty, defaultButton, doNotShowAgain);
+      }
+
+      private ViewResult showQuestionMessageBox(string message, IReadOnlyList<DialogResult> buttons, string yes, string no, string cancel, ViewResult defaultButton, Action doNotShowAgain)
```

Comment:
I think if you make this ```doNotShowAgain``` argument take a default argument of ```null``` then you would not need to add ```null``` args in method calls.

Reply:
You're right. Fixed in commit 1ad367850ff74c4378df84d90be4dac856d619db